### PR TITLE
ORCH-346 Add WS mandatory fields to orchestrator/reset

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'mavenplugin' package (org.sonarsource.orchestrator:orchestrator-maven-plugin:target/generated-sources/plugin/com/sonar/orchestrator/mavenplugin) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'mavenplugin' package (org.sonarsource.orchestrator:orchestrator-maven-plugin:target/generated-sources/plugin/com/sonar/orchestrator/mavenplugin) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

File: sonar-orchestrator/plugins/sonar-reset-data-plugin/src/main/java/com/sonar/orchestrator/resetdataplugin/ResetDataWebService.java [hunk 1]
```diff
@@ -21,10 +21,10 @@ public void define(Context context) {
 
     controller.createAction("reset")
       .setDescription("Reset data")
+      .setSince("Orchestrator 3.4")
       .setInternal(true)
       .setPost(true)
       .setHandler(new RequestHandler() {
-        @Override
```
Comment (simonbrandhof) [inline sonar-orchestrator/plugins/sonar-reset-data-plugin/src/main/java/com/sonar/orchestrator/resetdataplugin/ResetDataWebService.java]: `@Override` should not be removed.

**Comment by simonbrandhof** [comment]
LGTM

**Pull request closed without merging**